packages/marko-web-theme-default/components/site-footer/index.marko

```diff
@@ -1,6 +1,7 @@
 $ const { config, site } = out.global;
 
 $ const blockName = input.blockName || "site-footer";
+$ const showCopyright = input.showCopyright !== undefined ? input.showCopyright : true;
```

Comment:
Use the `defaultValue` utility from `@parameter1/marko-web-core/utils/default-value` for this